src/sonic-yang-models/tests/yang_model_tests/tests/dhcp_server_ipv4.json

```diff
@@ -67,8 +67,7 @@
         "eStr": ["Statement of 'ips' and 'ranges' cannot both exist"]
     },
     "DHCP_SERVER_IPV4_GATEWAY_ABSENT": {
-        "desc": "Missing gateway in DHCP_SERVER_IPV4",
-        "eStrKey" : "Mandatory"
+        "desc": "Missing gateway in DHCP_SERVER_IPV4"
```

Comment:
Do we need this test anyway? Why not just remove it?

Reply:
Removed